ckanext/datastore/logic/action.py

```diff
@@ -55,6 +55,10 @@ def datastore_create(context, data_dict):
     :type primary_key: list or comma separated string
     :param indexes: indexes on table (optional)
     :type indexes: list or comma separated string
+    :param triggers: triggers to apply to this table, eg: [
+        {"function": "trigger_clean_reference"},
+        {"function": "trigger_check_codes"}]
+    :type triggers: list of dictionaries
```

Comment:
Good suggestion. You can use any postgresql function that returns type `trigger`, whether created by datastore_function_create or manually.